modules/http/http-worker.c

```diff
@@ -186,19 +198,44 @@ _append_auth_header(List *list, HTTPDestinationDriver *owner)
 }
 
 static void
-_collect_rest_headers(HTTPDestinationWorker *self)
+_set_error_from_slot_result(const gchar *signal,
+                            HttpHeaderRequestSlotResultType result,
+                            GError **error)
+{
+  switch (result)
+    {
+    case HTTP_HEADER_REQUEST_SLOT_CRITICAL_ERROR:
+      g_set_error(error, HTTP_HEADER_FORMAT_ERROR,
+                  HTTP_HEADER_FORMAT_SLOT_CRITICAL_ERROR,
+                  "Critical error during slot execution, signal:%s",
+                  signal_http_header_request);
+      break;
+    case HTTP_HEADER_REQUEST_SLOT_PLUGIN_ERROR:
+    default:
```

Comment:
The success branch is missing from this switch case. As a result: we will emit warning even if the connector was successful.

Reply:
Ah, indeed. thanks!